pages/common/shc.md

```diff
@@ -1,6 +1,7 @@
 # shc
 
 > Generic shell script compiler.
+> More information: <https://manned.org/shc>.
```

Comment:
Wondering if we should use manned here or point towards the actual git repo, as this is not a standard command. Although, we have used `manned` earlier IMO.
Thoughts ? @navarroaxel 
```suggestion
> More information: <https://github.com/neurobin/shc>.
```

Reply:
I begin to be very confused with the consistency rule on these links. Maybe I did not read some documentation about some rule?
What are priorization?
* source code?
* complete documentation?
* user-friendly doc?
* accurate, up-to-date documentation?

If someone has any pointers, hints or authority on this topic, I am very interested. 😄

Reply:
We had this discussion a time ago #5143, IMO I'm not sure about link the man page, but that's our first rule. If there is no man page, then the Git repo is a good source. I think the man page fits for this command.

Reply:
@mere-teresa I apologize, if there was a any confusion :(

Reply:
Thank you @navarroaxel 
And @mfrw it's not you, it was the context. No problem.